docs/common-reasons-for-ads-not-showing.mdx

```diff
@@ -0,0 +1,63 @@
+# Common reasons for ads not showing
+
+Use the following guide to understand common reasons why apps show few or no ads.
+
+## Did you recently create your AdMob account?
+
+Wait up to 24 hours after you create your account.
+When you first sign up for AdMob, your account is reviewed before it’s approved.
+This typically takes up to 24 hours, but in rare cases can take up to 2 weeks.
+You’ll be notified via email if your account is not approved by AdMob.  
+
+## Is your app or ad unit new?
+
+Wait up to one hour after you create an app or ad unit.
+In some cases, it can take a few days for ads to appear in new apps or ad units.
+Note: New iOS apps will not show Google ads until they’re listed in the Apple App Store.
+Newly registered apps and newly created ad units typically take up to an hour to activate.
+Because of this, you may not see live impressions immediately.
```

Comment:
Is the phrase "live impressions" the same as "ads"? Just curious, I'm not familiar with the terminology they use in upstream documentation

```suggestion
New apps and ad units do not show live impressions immediately.
Here are common reasons you may not be receiving live impressions:
- It usually takes at least an hour after you create an app or ad unit
- In some cases **(what cases?)**, it can take a few days for ads to appear in new apps or ad units
- New iOS apps will not show Google ads until they’re listed in the Apple App Store.
```

Reply:
Impressions are how many times your ads are viewed by different users.